### ISSUE TITLE
Vaccinations can be neither created nor updated

### ISSUE THREAD

**Issue by anna-otrokhova** [PulseTile/PulseTile#58]
After solving the issue https://github.com/ethercis/ethercis/issues/72#issuecomment-343370833
We can check creation and editing of Vaccination heading

I will open the issue here to test it afterwards

This issue is related to this one https://github.com/PulseTile/PulseTile-React/issues/65

**Comment by anna-otrokhova** [comment]
For now, Vaccinations can be only created, editing can't be checked till source id is added to each PUT request. Once it is done we will check. 

Related issue: https://github.com/PulseTile/PulseTile/issues/54

**Comment by anna-otrokhova** [comment]
I have checked the editing Vaccination on dev (Angular box) and showcase and got errors. Here is dev box:
1) Request http://prntscr.com/hjdsyv
2) Response http://prntscr.com/hjdspn
3) server log http://prntscr.com/hjdtd7

**Comment by anna-otrokhova** [comment]
No one heading can be updated because of lack of configurations for showcase box on back-end side. That is not an issue of Vaccination heading.  There is another issue for this https://github.com/RippleOSI/Ripple-Qewd/issues/69
